substrate/client/transaction-pool/src/graph/pool.rs

```diff
@@ -358,7 +373,8 @@ impl<B: ChainApi, L: EventHandler<B>> Pool<B, L> {
 			}
 		}
 
-		log::debug!(target: LOG_TARGET,"prune: validated_counter:{validated_counter}, took:{:?}", now.elapsed());
+		let known_txs_count = known_provides_tags.map(|inner| inner.len()).unwrap_or(0);
+		log::debug!(target: LOG_TARGET,"prune: validated_counter:{validated_counter}, known_txs_count: {known_txs_count}, unknown_txs_count: {unknown_txs_count}, reused_txs_count: {reused_txs_count}, took:{:?}", now.elapsed());
```

Comment:
Please use tracing like formatting:
```suggestion
		log::debug!(target: LOG_TARGET, validated_counter, known_txs_count, unknown_txs_count, reused_txs_count, duration= now.elapsed(),"prune");
```

Reply:
Actually, we need this: https://github.com/paritytech/polkadot-sdk/pull/8001 to be able to use structured logging. log crate doesn't support it. Will update after https://github.com/paritytech/polkadot-sdk/pull/8001 gets merged.

Reply:
Resolved here: [46ddcda](https://github.com/paritytech/polkadot-sdk/pull/7980/commits/46ddcdabf1b0635c16be00231de2aa7ccd560446).